packages/scss/src/components/_progress.scss

```diff
@@ -12,6 +12,7 @@
 	left: 0;
 	position: absolute;
 	top: 0;
+	transition: background _theme("commons.animations.durations.standard") ease;
```

Comment:
Would it be useful to have a `width` transition, in case of a sudden surge in width (going suddenly from 0 to 15% for example) ?